doc/man/zcash-cli.1

```diff
@@ -3,7 +3,7 @@
 .SH NAME
 zcash-cli \- manual page for zcash-cli v1.0.7
 .SH DESCRIPTION
-Zcash RPC client version v1.0.7
+Zcash RPC client version v1.0.7\-24a24abf7\-dirty
```

Comment:
Hey, this is a classic antipattern of committing the output of some automated generator process. Let's not bother with changing that for now, but is there any reason we should not `git rm` these files and generate them on demand in future releases?